GNOME #448173, Debian #468130; use python-config to set PYTHON_INCLUDES; fixes
support for -dbg package

Index: pygobject-2.14.2/configure.ac
===================================================================
--- pygobject-2.14.2.orig/configure.ac	2008-06-07 14:06:35.211005448 +0200
+++ pygobject-2.14.2/configure.ac	2008-06-07 14:06:41.314510224 +0200
@@ -74,7 +74,11 @@ AM_CHECK_PYTHON_HEADERS(,[AC_MSG_ERROR(c
 
 AC_MSG_CHECKING([for PySignal_SetWakeupFd in Python.h])
 py_prefix=`$PYTHON -c "import sys; print sys.prefix"`
+if test -x "$PYTHON-config"; then
+PYTHON_INCLUDES=`$PYTHON-config --includes 2>/dev/null`
+else
 PYTHON_INCLUDES="-I${py_prefix}/include/python${PYTHON_VERSION}"
+fi
 old_CPPFLAGS=$CPPFLAGS
 CPPFLAGS="-Wall -Werror $PYTHON_INCLUDES"
 AC_TRY_COMPILE([#include <Python.h>],
